my-app/src/app/screens/Game/index.js

```diff
@@ -1,47 +1,24 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 
 import calculateWinner from '../../utils/utils';
+import { actions } from '../../../redux/game/actions';
 
 import Board from './components/Board';
 
 class Game extends Component {
-  state = {
-    history: [{ squares: Array(9).fill(null) }],
-    xIsNext: true,
-    stepNumber: 0
-  };
-
   handleClick = i => {
-    const history = this.state.history.slice(0, this.state.stepNumber + 1);
-    const current = history[history.length - 1];
-    const squares = current.squares.slice();
-    if (calculateWinner(squares) || squares[i]) {
-      return;
-    }
-    squares[i] = this.state.xIsNext ? 'X' : 'O';
-    this.setState({
-      history: history.concat([
-        {
-          squares: squares
-        }
-      ]),
-      stepNumber: history.length,
-      xIsNext: !this.state.xIsNext
-    });
+    this.props.dispatch({ type: actions.handleClick, pos: i });
   };
 
-  jumpTo = step => {
-    this.setState({
-      stepNumber: step,
-      xIsNext: step % 2 === 0
-    });
+  jumpTo = stepNumber => {
+    this.props.dispatch({ type: actions.jumpTo, stepNumber });
   };
```

Comment:
I'd move `jumpTo` to `mapDispatchToProps`. What do you think ?
Remember that, when using `mapDispatchToProps`, `dispatch` is not being received via props anymore unless we tell `connect` to do so

Reply:
Also, I'd use `payload` convention here too